community/catalog/compromises/2020/thegreatsuspender.md

```diff
@@ -20,10 +20,12 @@ diverged from its Github source. A minor change in the manifest was now being
 shipped on the chrome web store, which was not included in Github. This is a
 major concern.
 
+<!-- markdown-link-check-disable -->
 As a final red flag, no part of the web-store posting has been updated to account
 for this. [@greatsuspender](https://github.com/greatsuspender) remains listed as
-the maintainer, and the privacy policy makes no mention of the new tracking or
+the maintainer, and a privacy policy makes no mention of the new tracking or
```

Comment:
What's happening with this word change? It seems like an impactful tweak, considering that it shifts from discussing the policy for the linked repo to implying that many policies may exist. Better to leave this for a suggested change that the WG can respond to.

Reply:
The page doesn't exist anymore and the repository doesn't have a privacy policy. I adjusted enough to say there was a policy but that policy isn't *the* policy.

Reply:
This effort is part of addressing your feedback highlighted during the approval discussion of the IA document, specifically regarding 'broken links throughout the repo that we'll need to find and clean up,' but fine by me either way. 

The wording probably should be a 'privacy policy that existed for a period of time' or 'at the time.' I tried Wayback Machine to replace it with that but it wasn't able to pull it

Reply:
Ah, frick. I see how you got there. 

Can we get a thumbs up from the WG lead or last committer? If we don't hear back, I'd suggest its better to leave with a broken link than update it with a commit that may not improve the doc.